crmd/throttle.c

```diff
@@ -102,14 +83,16 @@ int throttle_num_cores(void)
  *       This will return NULL if the daemon is being run via valgrind.
```

Comment:
Wanted to highlight that's nothing granted for non-root processes
and we should deal with it.

It's actually quite sane not prevent users inspection of foreign
processes, it's not their business.